source/pdo_sqlsrv/config.m4

```diff
@@ -1,68 +1,68 @@
-PHP_ARG_WITH(pdo_sqlsrv, for pdo_sqlsrv support,

```

Comment:
I can only surmise that upon executing 'git push' the line endings did not get changed to CRLF. I fixed it by downloading the original and uploading it back using the web site instead.